pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1099,6 +1099,18 @@ let
         };
       };
 
+      csharpier.csharpier-vscode = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "csharpier-vscode";
+          publisher = "csharpier";
+          version = "1.7.3";
+          hash = "sha256-/ZLjnlLl6xmgEazdCbnuE6UuuV1tDwAjpxz+vmBuYHE=";
+        };
+        meta = {
```

Comment:
Meta is incomplete. Add changelog, downloadPage, description, homepage, maintainers, etc.
Consider making yourself a maintainer to be notified on updates.